pinot-common/src/main/java/org/apache/pinot/common/config/GrpcConfig.java

```diff
@@ -0,0 +1,80 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pinot.common.config;
+
+import com.google.common.collect.ImmutableMap;
+import java.util.Map;
+import org.apache.pinot.common.utils.TlsUtils;
+import org.apache.pinot.spi.env.PinotConfiguration;
+
+
+public class GrpcConfig {
```

Comment:
Is this used for both grpc server and client, are those fields common?
Shall we separate them to GrpcServerConfig/GrpcClientConfig

Reply:
Should be ok to share

Reply:
I'd suggest keeping them separate because even though some of the config can be shared, logically they should be separate config

Reply:
Sure I can create 2 subclass

Reply:
actually i think we should just keep it as one. since 
--> pinot configuration is passed in and uses PREFIX to parse tls config; 
--> tls config is also using same config file for both client side and server side. 

let's make it consistent.